src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -116,14 +116,30 @@ private class NextBackup
                 lastIncrementalBackup, configuration, skipErrorLog: skipErrorLog);
 
             if (nextFullBackup == null && nextIncrementalBackup == null)
+            {
```

Comment:
Failure to parse the cron expression for both full and incremental backups.
I'll move this to the "save the configuration" endpoint